RWBY_RoC-Unofficial_Tabletop_RPG/RWBY-UnofficialTabletopRPG.css

```
@@ -153,6 +315,30 @@ input[type=number] {
     -moz-appearance: textfield;
 }
 
+input[type=number].sheet-roc::-webkit-inner-spin-button,
+input[type=number].sheet-roc::-webkit-outer-spin-button {
+    -webkit-appearance: enable;
+    position: relative;
+    right: -1px;
+    width: 0.8em;
+    z-index: 5;
+    opacity: 0.5;
+    background: url('https://s3.amazonaws.com/files.d20.io/images/70896407/88_Q0INDLeVm11tVnU4T-A/max.jpg?1547165196') center center;
```

```suggestion
    background: url('https://raw.githubusercontent.com/Roll20/roll20-character-sheets/master/RWBY_RoC-Unofficial_Tabletop_RPG/webkit.jpg') center center;
```
This will be the proper url for the image when your changes becomes part of the Roll20 repo.

---

Oh, Thank you for your comments! I'll try it when the pull is accepted!